app/pages/collections/list.cjsx

```diff
@@ -64,24 +132,36 @@ List = React.createClass
             </div>
             <nav>
               {if meta
+                buttonClasses = classNames {
+                  "pill-button": true
+                  "project-pill-button": @props.project?
+                }
                 <nav className="pagination">
                   {for page in [1..meta.page_count]
                     active = (page is +location.query.page) or (page is 1 and not location.search)
                     <Link
                       key={page}
                       to={"#{@props.location.pathname}?page=#{page}"}
-                      activeClassName="active"
-                      className="pill-button"
+                      activeClassName= "active"
+                      className={buttonClasses}
                       style={border: "2px solid" if active}>
```

Comment:
Barf.  Yeah, I was misreading `{page}` as an attribute.  That `>` is necessary there.